### DISCUSSION TITLE
make widget move around cornor

### DISCUSSION THREAD

**Discussion by Harshjeet** [dailydotdev/daily#1598] [Feature Requests]
sometime it is very difficult to navigate some page as fixed widget blocked some functionalities of the web pages. I request to the developer please make it flexible so that we can drag it different position of screen's corner.

**Comment by idoshamun** [comment]
Hi @Harshjeet,

It's already reported here https://github.com/dailydotdev/daily/discussions/622.
At the moment it's not prioritized
